spec.go

```diff
@@ -116,3 +125,50 @@ var specCommand = cli.Command{
 		fmt.Printf("%s", data)
 	},
 }
+
+func (m*Mount)UnmarshalJSON(s []byte) error {
+	var err error = nil
+	var rm readMount
+	var v interface{}
+
+	err = json.Unmarshal(s,&v)
+
+	if err != nil {	/* general json error */
+		return err
+
+	} else {
```

Comment:
It adds too much indentation and makes code harder to read (you have to parse more blocks than necessary). But since this isn't meant to be merged, it's of little consequence.